ddf-rpc-api/pom.xml

```diff
@@ -0,0 +1,292 @@
+                            platform-util,
+                            log-sanitizer
+                        </Embed-Dependency>
+                        <Import-Package>
+                            !com.google.common.util.concurrent.internal,
+                            !com.google.errorprone.annotations.*,
+                            !javax.annotation,
+                            !org.checkerframework.checker.nullness.qual,
+
+                            ddf.action.impl;version="[2.19,3)",
+                            ddf.catalog.content.data;version="[2.19,3)",
+                            ddf.catalog.content.operation;version="[2.19,3)",
+                            ddf.catalog.data.types.experimental;version="[2.19,3)",
+                            ddf.catalog.data.types;version="[2.19,3)",
+                            ddf.catalog.data;version="[2.19,3)",
+                            ddf.catalog.endpoint;version="[2.19,3)",
+                            ddf.catalog.event;version="[2.19,3)",
+                            ddf.catalog.federation;version="[2.19,3)",
+                            ddf.catalog.filter;version="[2.19,3)",
+                            ddf.catalog.operation;version="[2.19,3)",
+                            ddf.catalog.plugin;version="[2.19,3)",
+                            ddf.catalog.resource;version="[2.19,3)",
+                            ddf.catalog.security;version="[2.19,3)",
+                            ddf.catalog.source;version="[2.19,3)",
+                            ddf.catalog.transform;version="[2.19,3)",
+                            ddf.catalog.util;version="[2.19,3)",
+                            ddf.catalog.validation.report;version="[2.19,3)",
+                            ddf.catalog.validation.violation;version="[2.19,3)",
+                            ddf.catalog.validation;version="[2.19,3)",
+                            ddf.catalog;version="[2.19,3)",
+                            org.codice.ddf.configuration;version="[2.19,3)",
+
+                            org.apache.commons.lang3.tuple;version="[3.8.0,4.0.0)",
+                            org.apache.commons.lang3;version="[3.8.0,4.0.0)",
+
+                            org.apache.shiro.subject;version="[1.4,2)",
+                            org.apache.shiro;version="[1.4.0,2.0.0)",
+
+                            org.opengis.annotation;version="[19.1.0,21.0.0)",
+                            org.opengis.filter.expression;version="[19.1.0,21.0.0)",
+                            org.opengis.filter.sort;version="[19.1.0,21.0.0)",
+                            org.opengis.filter;version="[19.1.0,21.0.0)",
+                            *
+                        </Import-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>com.coveo</groupId>
+                <artifactId>fmt-maven-plugin</artifactId>
+                <version>2.3.0</version>
```

Comment:
✏️ DDF is on 2.9.1, but if you want to leave a reformat to another PR then that's ok

Reply:
Yup i'll update and include it in the next PR (#33)